stock_archive_constraint/models/product_template.py

```diff
@@ -1,12 +0,0 @@
-# Copyright 2020 Tecnativa - Víctor Martínez
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
-
-from odoo import api, models
-
-
-class ProductTemplate(models.Model):
-    _inherit = "product.template"
-
-    @api.constrains("active")
-    def _check_active_stock_archive_constraint(self):
-        self.product_variant_ids._check_active_stock_archive_constraint()
```

Comment:
Why this one has been removed? Is this working the same if you try to archive the template?

Reply:
This file was never used and is not imported in `__init__.py`

Reply:
> Why this one has been removed? Is this working the same if you try to archive the template?

This file was never used and is not imported in __init__.py
And, yes, it working from product.product and product.template